### ISSUE TITLE
Rename 'Configure site info and Logo'

### ISSUE THREAD

**Issue by PeaceGeeks** [PeaceGeeksSociety/amani#799]
Once #798 is done, rename 'Configure Site Info and Logo' to 'Manage Logo & Tagline'

**Comment by MagdiRizkallah** [comment]
There is more to that than just Logo & Tagline!!! e.g. address, phone, email, work hours, location, etc,,,

**Comment by PeaceGeeks** [comment]
I am moving those fields to a 'Contact Settings' page which is more intuitive. There is a separate ticket about this.

**Comment by akochlan** [comment]
Validated it as Amani Admin. 'Configure Site Info and Logo' has been renamed to 'Manage Logo & Tagline'. Passed and Closed. 
![manage logo tagline](https://cloud.githubusercontent.com/assets/16566072/12152652/c1c7f034-b483-11e5-8fa1-9fc482b74ce5.jpg)

**Comment by PeaceGeeksProjects** [comment]
Tx @akochlan.  However, this is for the standard theme, not the Amank Zen theme.

**Comment by akochlan** [comment]
@MagdiRizkallah However, this is for the standard theme, not the Amank Zen theme. What did you mean?

**Comment by akochlan** [comment]
Logged as superadmin, select Appearance from top menu.
Amani Zen  - default theme, see screenshot.
![default theme](https://cloud.githubusercontent.com/assets/16566072/12175550/e4872088-b530-11e5-95df-57c16c9961ac.png)

**Comment by PeaceGeeksProjects** [comment]
@akochlan this issue is NOT labelled Amani Zen Theme...  We do not need to test it

**Comment by akochlan** [comment]
@PeaceGeeksProjects  tested accidentally bc it has High Priority

**Comment by bensinjin** [comment]
Done and deployed to dev.amani-zen-theme.peacegeeks.org & dev.libya.peacegeeks.org.

**Comment by bensinjin** [comment]
@PeaceGeeksProjects please confirm and close.

**Comment by PeaceGeeksProjects** [comment]
@bensinjin - can you move compelted tix to the Done pipeline?




*Renee Black*,
*Executive Director*
#210 - 128 West Hastings Street
Vancouver, BC, V6B 1G8
*p.* 778-235-7458 | *e.* renee@peacegeeks.org
*skype**.* reneeontheroad
*pgp. *9253 3C6E 70B4 5211 6854 ED5E B585 C32D 264C 9CB6

**Comment by bensinjin** [comment]
will do, thx

**Comment by PeaceGeeksProjects** [comment]
@bensinjin Howewver, this was not done...  remember it needs #798 completed.  It now goes to http://dev.amani-zen-theme.peacegeeks.org/admin/config/system/site-information which does not have the Logo and Favicon fields and has some weird fields.

Moving back to In Progress.

**Comment by bensinjin** [comment]
@PeaceGeeksProjects This is what I've Implemented these buttons:
Manage Logo & Tagline ->  Site Name, Tagline, Logo
Manage Contact Page -> Address, Lat/Long, Email, Phone

Deployed to dev. Moving to Done.

**Comment by PeaceGeeksProjects** [comment]
Verified on Dev & LYM
